packages/devtools_app/lib/src/debugger/evaluate.dart

```diff
@@ -202,7 +201,7 @@ String _valueAsString(InstanceRef ref) {
     return ref.valueAsString;
   }
 
-  if (ref.valueAsStringIsTruncated == true) {
+  if (ref.valueAsStringIsTruncated) {
```

Comment:
I think we do want to keep the `ref.valueAsStringIsTruncated == true` check here, to guard against the valueAsStringIsTruncated field being null (we have had exceptions from this in the past).

Reply:
sounds good, will revert!